Civi/Api4/Log.php

```diff
@@ -0,0 +1,22 @@
+<?php
+/*
+ +--------------------------------------------------------------------+
+ | Copyright CiviCRM LLC. All rights reserved.                        |
+ |                                                                    |
+ | This work is published under the GNU AGPLv3 license with some      |
+ | permitted exceptions and without any warranty. For full license    |
+ | and copyright information, see https://civicrm.org/licensing       |
+ +--------------------------------------------------------------------+
+ */
+namespace Civi\Api4;
+
+/**
+ * Log
+ *
+ * @since 5.45
+ * @package Civi\Api4
+ */
+class Log extends Generic\DAOEntity {
+  use Generic\Traits\ReadOnly;
```

Comment:
This trait has been renamed:
```suggestion
  use Generic\Traits\ReadOnlyEntity;
```